### PR TITLE
Updated the login feature on welcome site.

### PR DESCRIPTION
**Pull request by Rucha1499**
Added some css and html properties for correct alignment of login button on welcome site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,8 +23,8 @@
         <li><a href="faq.html">FAQ</a></li>
         <li><a href="discord.html">Discord</a></li>
       </ul>
-      <div class="login">
-        <a href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97" id="login-btn">Login</a>
+      <div id="login-btn">
+        <a href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97">Login<a>
```
Comment (swarajpure) [inline index.html:27]: The anchor tag has not been closed properly. Please fix it.

**Review by swarajpure — CHANGES_REQUESTED** [review]
An anchor tag has not been closed properly. Please fix it.

File: css/pages/navbar.css [hunk 2]
```diff
@@ -21,6 +21,10 @@ nav {
 #login-btn {
   float: right;
```
Comment (swarajpure) [inline css/pages/navbar.css:22]: Are you sure if we need `float: right` and `text-transform: capitalize`?

There's no difference even if disable them:

image

**Review by swarajpure — COMMENTED** [review]
Two CSS properties are having no effect even after disabling them. Please check if we really need them. Thank you!

File: css/pages/navbar.css (hunk 2 above)
Comment (Rucha1499) [reply css/pages/navbar.css:22]: Yes! Not actually. Thanks!

**Review by swarajpure — APPROVED** [review]
Looks good to me. Thank you! 😄

**Pull request merged**